curvefs/src/metaserver/copyset/apply_queue.h

```diff
@@ -63,8 +67,11 @@ class CURVE_CACHELINE_ALIGNMENT ApplyQueue {
     void StartWorkers();
 
     struct TaskWorker {
-        explicit TaskWorker(size_t cap)
-            : running(false), worker(), tasks(cap) {}
+        TaskWorker(size_t cap, std::string workerName)
```

Comment:
Why remove explicit

Reply:
in most cases, `explicit` should only mark single argument constructor